src/librustdoc/html/static/playpen.js

```diff
@@ -15,6 +15,11 @@ document.addEventListener('DOMContentLoaded', function() {
     'use strict';
 
     if (!window.playgroundUrl) {
+        var run_buttons = document.querySelectorAll(".test-arrow");
```

Comment:
I think there is some value in caring about keeping consistent with naming conventions. It's not clear to me why we shouldn't care about JavaScript.